feat(samples): add fontawesome icon sample

BF150 opened the pull request:
## Summary

Internal sample addition — added a new sample demonstrating how to use Font Awesome icons with KoliBri components.

## Changes

- Added Font Awesome icon integration sample to the sample app

---

<details>
<summary>🇩🇪 Deutsche Zusammenfassung</summary>

## Zusammenfassung

Interne Sample-Erweiterung — neues Beispiel hinzugefügt, das die Verwendung von Font-Awesome-Icons mit KoliBri-Komponenten demonstriert.

## Änderungen

- Font-Awesome-Icon-Integrations-Sample zur Beispiel-App hinzugefügt

</details>

BF150 committed:
added sample$
$
Refs: #8692

deleonio commented on packages/samples/react/src/components/icon/font-awesome.tsx:
```diff
@@ -0,0 +1,18 @@
+export const IconFontAwesome: FC = () => (
+	<>
+		<SampleDescription>
+			<p>KolIcon renders FontAwesome icons, if you have installed this font.</p>
```
Kleine Anmerkung hier: wenn sie im Theme hinterlegt ist. Nimm unbedingt die Version dazu, damit es später keine Missverständnisse gibt.

In der sample app brauchen wir wahrscheinlich auch sowas inject assets.

BF150 replied:
Hab den Text angepasst. 
Du meintest ja das default theme braucht die codicon icons gar nicht in den assets, deswegen hab ich die entfernt, aber im build gibts jetzt deswegen ne warning. Und der Angular build schlägt fehl wegen der bundle size. Ich denke mal weil er jetzt die font awesome icons holt. Soll ich die maximale bundle size vergrößern?
https://github.com/public-ui/kolibri/actions/runs/19231347261/job/54970424205?pr=8811#step:4:504
<img width="944" height="353" alt="image" src="https://github.com/user-attachments/assets/6cf3df04-25c4-451b-8ba1-99f7b9dcfea4" />

deleonio replied:
Ich denke, wir müssen das einfach erhöhen. Es handelt sich hier auch nur um einen Test-Ballon.

BF150 replied:
Jetzt hat das Budget viel Puffer 😄

deleonio reviewed (commented):
.

BF150 committed:
added fa & fixed icon component$
$
Refs: #8692

BF150 committed:
Update all snapshots$
$
Refs: #8692

BF150 committed:
changed snapshot sizes$
$
Refs: #8692

BF150 committed:
Merge branch 'task/8692-fa-example' of https://github.com/public-ui/kolibri into task/8692-fa-example

BF150 committed:
Merge branch 'release/3' into task/8692-fa-example

BF150 committed:
Update all snapshots$
$
Refs: #8692

deleonio committed:
Add Font Awesome 6.1.1 assets including CSS and webfonts

- Added v4-shims.min.css for Font Awesome 6.1.1 compatibility.
- Introduced v5-font-face.css and v5-font-face.min.css for Font Awesome 5 font definitions.
- Included webfont files for Font Awesome brands, regular, and solid styles in both .ttf and .woff2 formats.
- Added v4 compatibility webfonts for backward compatibility.

Refs: #8692

BF150 committed:
angular budgets erhöht$
$
Refs: #8692

BF150 committed:
Merge branch 'release/3' into task/8692-fa-example

BF150 committed:
update snappies$
$
Refs: #8692
